submissions/fiyingabriel/htmlForms/contactUs.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+    <h2>Subscribe to get the latest info on our products</h2>
+     <form action="#" method="post" enctype="multiort/form-data">
```

Comment:
Remove the attributes `action`, `method`, `post` and `enctype` from the form tag